src/Aquifer.API/Endpoints/Projects/Get/Endpoint.cs

```diff
@@ -17,7 +17,7 @@ public override void Configure()
 
     public override async Task HandleAsync(Request req, CancellationToken ct)
     {
-        if (userService.HasPermission(PermissionName.ReadProjectsInCompany) && !await HasSameCompanyAsProject(req.ProjectId, ct))
+        if (!userService.HasPermission(PermissionName.ReadProject) && !await HasSameCompanyAsProject(req.ProjectId, ct))
```

Comment:
The logic was reversed here which meant that if the user had both `ReadProjectsInCompany` and `ReadProject` then they would be restricted as if they only had `ReadProjectsInCompany`.